treon/treon.py

```diff
@@ -87,21 +104,21 @@ def print_test_collection(notebooks):
               -----------------------------------------------------------------------\n""")
     message += '\n'.join(notebooks) + '\n'
     message += '-----------------------------------------------------------------------\n'
-    print(message)
+    LOG.debug(message)
 
 
 def get_notebooks_to_test(args):
     path = args['PATH'] or os.getcwd()
     result = []
 
     if os.path.isdir(path):
-        print('Recursively scanning {path} for Notebooks...'.format(path=path))
+        LOG.info('Recursively scanning %s for notebooks...', path)
         path = os.path.join(path, '')  # adds trailing slash (/) if it's missing
         glob_path = path + '**/*.ipynb'
         result = glob.glob(glob_path, recursive=True)
     elif os.path.isfile(path):
         if path.lower().endswith('.ipynb'):
-            print('Testing notebook {path}'.format(path=path))
+            LOG.info('Testing notebook %s', path)
```

Comment:
This could also be Log.debug since when I run a single notebook test it looks something like this,

```
Executing treon version 0.1.0
Testing notebook /workspace/PythonDataScienceHandbook/notebooks/00.00-Preface.ipynb
Triggered test for /workspace/PythonDataScienceHandbook/notebooks/00.00-Preface.ipynb
```

Line 2 and 3 are essentially the same.